plutus-core/plutus-core/src/PlutusCore/Version.hs

```diff
@@ -48,6 +52,25 @@ instance Ord Version where
   compare (Version major1 minor1 patch1) (Version major2 minor2 patch2) =
     compare major1 major2 <> compare minor1 minor2 <> compare patch1 patch2
 
--- | The default version of Plutus Core supported by this library.
-defaultVersion :: Version
-defaultVersion = Version 1 0 0
+-- | The first version of Plutus Core supported by this library.
+firstVersion :: Version
+firstVersion = plcVersion110
+
+-- | Plutus Core version 1.0.0
+plcVersion100 :: Version
+plcVersion100 = Version 1 0 0
+
+-- | Plutus Core version 1.1.0
+plcVersion110 :: Version
+plcVersion110 = Version 1 1 0
+
+-- | The latest version of Plutus Core supported by this library.
+latestVersion :: Version
+latestVersion = plcVersion110
+
+instance Bounded Version where
```

Comment:
What is this useful for? Not all versions between `minBound` and `maxBound` are valid.

Reply:
I also think it's a bit suspicious now that I think about it. I think we want to say something like "you can enumerate the known versions of PLC". I think better versions might be:
1. `KnownVersion` newtype with `Enum` and `Bounded` instances
2. A `knownVersions :: Set Version` value

thoughts?

Reply:
I've added `knownVersions`.